File: challenges/4-fizz-buzz/typescript/marcosvag/fizzBuzz.ts
const input = parseInt(process.argv[2]);

function fizzBuzz(num:number):void {
    for(let e = 1; e <= num; e++) {
        if(!(e % 3) && !(e % 5)){
            console.log('FizzBuzz');
        }else if(!(e % 3)){
            console.log('Fizz');
        }else if(!(e % 5)){
            console.log('Buzz');
        }else {
            console.log(e);
        }
    }
    return
}

fizzBuzz(input)

File: challenges/4-fizz-buzz/python/nidomus/fizzbuzz.py

x = int(input('X: '))
print()
for i in range(1,x+1):
    
    if i%15==0:
        print('FizzBuzz')
        
    elif i%3 == 0:
        print('Fizz')
    elif i%5 ==0:
        print('Buzz')

    else:
        print(i)